src/musesounds/qml/MuseScore/MuseSounds/internal/MuseSoundsListView.qml

```diff
@@ -0,0 +1,129 @@
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.15
+
+import Muse.Ui 1.0
+import Muse.UiComponents 1.0
+import Muse.Extensions 1.0
+
+Column {
+    id: root
+
+    property alias model: view.model
+    readonly property alias count: view.count
+
+    property alias title: titleLabel.text
+
+    property var flickableItem: null
+    property int headerHeight: titleLabel.height + spacing
+
+    property NavigationPanel navigationPanel: NavigationPanel {
+        name: "MuseSoundsListView"
+        direction: NavigationPanel.Both
+        accessible.name: root.title
+    }
+
+    spacing: 16
+
+    signal getSoundRequested(var soundId)
+    signal navigationActivated(var itemRect)
+
+    function focusOnFirst() {
+        // Force the view to load the items. Without this, `view.itemAtIndex(0)` might be null even when `count > 0`,
+        // causing navigation to break when calling this function from `resetNavigationFocus()` in `PluginsPage`.
+        view.forceLayout()
+
+        view.itemAtIndex(0).requestActive()
+    }
+
+    StyledTextLabel {
+        id: titleLabel
+        width: parent.width
+        font: ui.theme.headerFont
+        horizontalAlignment: Text.AlignLeft
+    }
+
+    GridView {
+        id: view
+
+        readonly property int columns: Math.max(0, Math.floor(width / cellWidth))
+
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.leftMargin: -spacingBetweenColumns / 2
+        anchors.rightMargin: -spacingBetweenColumns / 2
+
+        height: contentHeight - spacingBetweenRows
+
+        // We don't want this GridView to be scrollable; it would conflict with the containing Flickable.
+        interactive: false
+        highlightFollowsCurrentItem: false // prevent automatic scrolling
+
+        readonly property real spacingBetweenColumns: 22
+        readonly property real spacingBetweenRows: 16
+
+        readonly property real actualCellWidth: 350
+        readonly property real actualCellHeight: 150
+
+        cellWidth: actualCellWidth + spacingBetweenColumns
+        cellHeight: actualCellHeight + spacingBetweenRows
+
+        delegate: Item {
+            id: item
+
+            height: view.cellHeight
+            width: view.cellWidth
+
+            function requestActive() {
+                _item.navigation.requestActive()
+            }
+
+            MuseSoundItem {
+                id: _item
```

Comment:
This is a rather atypical ID format for a delegate. Maybe just item? Or soundItem